vendor/github.com/siddontang/go-mysql/canal/sync.go

```diff
@@ -13,11 +13,12 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+//  m  多行模式：让 ^ 和 $ 匹配整个文本的开头和结尾，而非行首和行尾(默认为 false)
```

Comment:
please split the PR and send this to the go-mysql project, not here.

Btw, please use English comments.